### DISCUSSION TITLE
中药、药经、医书感觉也可以安排一波

### DISCUSSION THREAD

**Discussion by baiyumg** [hefengbao/jingmo#63] [Ideas]
如题，有些医书药经啥的也很有意思，不过译文偏少，网上大部分找到的也都是原文，阅读难度较大。不过一味味中药，作为闲时浏览，如果能有一些概述、解释，个人认为也很棒。
希望大佬能够安排上。
膜拜，感恩！

**Comment by hefengbao** [comment]
我也是从网上找的数据

**Reply by baiyumg** [reply]
明白，搜集数据还挺麻烦的。
我自己也去找找看看。
这项目做的太棒了，哈哈哈。
再次感谢！
